crates/router/src/core/webhooks.rs

```diff
@@ -783,7 +776,7 @@ pub async fn create_event_and_trigger_outgoing_webhook<W: types::OutgoingWebhook
                     logger::error!(error=?err, event=?webhook_event, "Error Logging Outgoing Webhook Event");
                 }
             }
-        });
+        }.in_current_span());
```

Comment:
I think this change is addressing your [PR](https://github.com/juspay/hyperswitch/pull/3590/files) @sahkal